### ISSUE TITLE
SPEC is slow to get

### ISSUE THREAD

**Issue by vincele** [vegastrike/Vega-Strike-Engine-Source#272]
Is there a way to get SPEC acceleration faster?

It takes a very long time to reach the 97C max velocity (and even that is too slow for in system flight)

I find myself staring at an almost IDLE screen with nothing to do but wait for my ship to get from a base to another. Gett ing from base to base is taking on the order of a quarter of an hour (IRL), which is painful.

What am I missing ?

Where in the code should I poke to try to get something less realistic but more gamey ?

I already tried a few things here & there, but nothing changed...

**Comment by evertvorster** [comment]
Hi, Vince. 

The theory behind SPEC is that it sort of contorts space and time to make it appear that you are traveling faster than light. This is why you have no momentum from these relavistic speeds. Now, this SPEC does not work in deep gravity wells, or near other objects. 

TL;DR, if you want SPEC to kick in quicker, get away from things in your vicinity quicker. 

I normally kick my ship into GCNT:TRAVEL mode by pressing "y", then pointing the nose away from planets and such. If I have an afterburner, I burn some fuel, too. That normally gets me into SPEC ranges, even with the aging Llama.

**Comment by vincele** [comment]
OK, I knew the theory behind SPEC, what I was missing is the GCNT:TRAVEL, this is making the thing better, thanks. Is this explained somewhere ?

**Comment by Loki1950** [comment]
In the pdf manual IIRC as for the reason for the two velocity modes in travel mode the average engagement of a combat encounter would be in the order of a few seconds to see/target/fire on an enemy.

**Comment by evertvorster** [comment]
Not sure if it's explained anywhere, however, GCNT:Maneuver is your ship's computer limiting top speed to ensure that you can maneuver safely.
Once you switch over to GCNT:Travel, you indicate that you prefer speed over maneuverability, and so it sets the top speed limit much higher. Keep in mind that in space any change in speed means burning fuel, so if you want a heavy ship to go fast, you will need a lot of fuel.
